### PR TITLE
[RELEASE] Release notes sanity check

### PR DESCRIPTION
**Pull request by ridz1208**
## Brief summary of changes
Release notes preparation for 23.0

@aces/loris_triage and @aces/loris_write feel free to suggest additional line items to this document, we will include them as necessary

@johnsaigle @driusan @cmadjar If you can check it out would be great

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,41 @@ core section.***
 - ***When possible please provide the number of the pull request(s) containing the 
 changes in the following format: PR #1234***
```
Comment (cmadjar) [inline CHANGELOG.md:9]: should probably remove the text above before release?

Comment (ridz1208) [reply CHANGELOG.md:9]: lines 1 to 9 ?

**Comment by ridz1208** [comment]
@cmadjar 
> I guess for this one, technically, this is still in the DB but not in the Menu tables, no?

No I think it's actually the module code that now defines the menu content. @driusan correct ?

File: CHANGELOG.md (hunk 1 above)
Comment (driusan) [reply CHANGELOG.md:9]: 3 to 9, the heading is still fine..

**Comment by ridz1208** [comment]
@driusan all changes done

**Pull request merged**